test/replication/gh-5430-cluster-mvcc.test.lua

```diff
@@ -32,19 +32,27 @@ test_run:cmd('create server replica2 with '..
 test_run:cmd('start server replica2')
 
 test_run:switch('master')
-box.error.injection.set('ERRINJ_WAL_DELAY', true)
+box.error.injection.set('ERRINJ_REPLICA_JOIN_DELAY', true)
 
 test_run:switch('replica1')
 _ = require('fiber').create(function() box.cfg{replication_anon = false} end)
 str = string.format('registering replica %s', box.info.uuid):gsub('-', '%%-')
-_ = test_run:wait_log('master', str)
+test_run:wait_log('master', str) ~= nil
 
 test_run:switch('replica2')
 _ = require('fiber').create(function() box.cfg{replication_anon = false} end)
+test_run:switch('replica2')
```

Comment:
Do you need this new switch here? You are already on `replica2` in this place.

Reply:
Sorry, it's a typo, dropped.